README.md

```diff
@@ -51,7 +51,9 @@ The end user chooses which hydrator to use. The default one should return domain
 
 ### Request builder
 
-The request builder will build request with multipart streams when necessary. 
+The request builder will build request with multipart streams when necessary. If 
```

Comment:
The request builder creates a PSR-7 request with a multipart stream when necessary